### ISSUE TITLE
Support non-standard ports / include :port in URL

### ISSUE THREAD

**Issue by apowers313** [danwrong/restler#220]
Currently supports HTTP on port 80 and HTTPS on port 443. It would be great to have an option to support HTTP / HTTPS on non-standard ports, or to force the inclusion of the :port parameter in the URL (http://foo.com:443) for testing.

The motivation for this is that [nock](https://www.npmjs.com/package/nock) currently errors out when used with restler because it is missing the :port part of the URL.

**Comment by mynameiswillporter** [comment]
I agree that i would be great to support non-standard ports.  Currently restler cannot support our dev-environment, as we have multiple http servers running on nonstandard ports during development.
